src/gcp/rules.ts

```diff
@@ -27,32 +27,69 @@ export async function getLatestRulesetName(
   projectId: string,
   service: string
 ): Promise<string | null> {
+  const releases = await listAllReleases(projectId);
+  const prefix = "projects/" + projectId + "/releases/" + service;
```

Comment:
Prefer template strings.

```suggestion
  const prefix = `projects/${projectId}/releases/${service}`;
```

Reply:
done